docs/usage.rst

```diff
@@ -391,6 +391,22 @@ would be a voice activity detection algorithm.
     idx = interface.process_file(files[0])
     idx
 
+Sometimes, it is required that a table (i.e., `pd.Series`
+or `pd.DataFrame`) is segmented and the ``labels`` from
+the original segments should be kept. For this,
+:class:`audinterface.Segment` has a dedicated method
+``process_table()``. This method is useful, if a
+segmentation (e.g., voice activity detection) is
+performed on an already labelled dataset in order
+to do data augmentation or teacher-student training,
+improving model performance for shorter chunks.
+
+.. jupyter-execute::
+
+    table = pd.DataFrame({"label": [i*2 for i in range(len(index))]}, index=index)
+    table_segmented = interface.process_table(table)
+    table_segmented
```

Comment:
Here, it might be easier to understand it when showing also the original table:

```suggestion
    table = pd.DataFrame({"label": [i*2 for i in range(len(index))]}, index=index)
    table
    
.. jupyter-execute::

    table_segmented = interface.process_table(table)
    table_segmented
```

But in order to make this work, you will have to adjust the `dataframe_to_html()` function at the top of `docs/usage.rst` as well:

```rst
.. Specify pandas format output in cells
.. jupyter-execute::
    :hide-code:
    :hide-output:

    import pandas as pd

    import audformat


    def dataframe_to_html(df_original):
        # Replace beginning of data path with ...
        df = df_original.copy()
        if len(df.index) > 0 and df.index.names[0] == "file":
            old_path = r".+/audb/emodb/1.3.0/d3b62a9b/wav/"
            new_path = r".../"
            if audformat.is_segmented_index(df.index):
                df.index = df.index.set_levels(
                    df.index.levels[0].str.replace(
                        old_path,
                        new_path,
                        regex=True,
                    ),
                    level=0,
                )
            else:
                df.index = df.index.str.replace(old_path, new_path, regex=True)

        return df.to_html(max_rows=6, max_cols=4)
```

The resulting HTML will then look like this:

![image](https://github.com/audeering/audinterface/assets/173624/4081879d-918b-4499-ad78-90cbca5ceaae)